config/engines.yml

```diff
@@ -249,6 +249,16 @@ rubymotion:
   enable_regexps:
   default_ratings_paths:
     - "**.rb"
+
```

Comment:
Nit, but right now we don't have spaces between engine definitions. Seems like a nice addition to me, but if we're going to do it I think we should do all of them at once.